packages/e2e-tests/specs/editor/plugins/image-size.test.js

```diff
@@ -30,11 +29,9 @@ describe( 'changing image size', () => {
 
 	it( 'should insert and change my image size', async () => {
 		await insertBlock( 'Image' );
-		await clickButton( 'Media Library' );
-
-		// Wait for media modal to appear and upload image.
-		await page.waitForSelector( '.media-modal input[type=file]' );
-		const inputElement = await page.$( '.media-modal input[type=file]' );
+		const inputElement = await page.waitForSelector(
```

Comment:
I couldn't tell why this test was failing. I've modified it to upload the image in a different way, using the image block placeholder's Upload button. This is actually more succinct, though reduces test coverage for the media modal.

It could be related to this core bug or other changes to the media modal - https://core.trac.wordpress.org/ticket/54411

Reply:
Though it might actually be better to have test coverage for the placeholder upload button, given there are other tests for using the media modal. I think it's worth keeping this change.